apps/src/code-studio/videos.js

```diff
@@ -47,11 +50,31 @@ function onVideoEnded() {
 }
 
 var currentVideoOptions;
-function onYouTubeIframeAPIReady() {
+window.onYouTubeIframeAPIReady = function () {
```

Comment:
YouTube's API requires this function to be in a global `window` context, and (I think) due to the ordering of operations inside of a function, if defined as a function this wouldn't actually be created on the initial video load (but would exist if the video was re-opened). Defining it this way ensures that it exists early enough.

This file is a good candidate to circle back to and refactor.

Reply:
This is a super useful comment. Mind including something similar as a comment in the code too?

Reply:
Sure thing. Done.